code/game/turfs/closed/wall/reinf_walls.dm

```diff
@@ -50,19 +50,22 @@
 		playsound(src, 'sound/effects/bang.ogg', 50, 1)
 		to_chat(M, "<span class='warning'>This wall is far too strong for you to destroy.</span>")
 
-/turf/closed/wall/r_wall/try_destroy(var/obj/item/pickaxe/drill/I, mob/user, turf/T)
+/turf/closed/wall/r_wall/try_destroy(obj/item/I, mob/user, turf/T)
 	if(istype(I, /obj/item/pickaxe/drill/jackhammer) || istype(I, /obj/item/pickaxe/drill/diamonddrill) || istype(I, /obj/item/pickaxe/drill/cyborg/diamond))
+		var/obj/item/pickaxe/drill/drl = I
 		to_chat(user, "<span class='notice'>You begin to smash though [src]...</span>")
-		if(do_after(user, 14*I.drill_delay, 1, target = src))
+		if(do_after(user, 14*drl.drill_delay, 1, target = src))
+
 			if(!istype(src, /turf/closed/wall/r_wall))
```

Comment:
Czy to jest potrzebne skoro to jest funkcja klasy `/turf/closed/wall/r_wall` i nie powinna być wywoływana przez obiekty innych klas?